core/src/main/scala/org/apache/spark/ui/storage/StoragePage.scala

```diff
@@ -30,11 +30,32 @@ private[ui] class StoragePage(parent: StorageTab) extends WebUIPage("") {
   private val listener = parent.listener
 
   def render(request: HttpServletRequest): Seq[Node] = {
-    val rdds = listener.rddInfoList
-    val content = UIUtils.listingTable(rddHeader, rddRow, rdds, id = Some("storage-by-rdd-table"))
+    val blocks = listener.allBlocks.sortBy(_.blockId.toString)
+    val streamBlocks = blocks.filter(_.blockId.isInstanceOf[StreamBlockId])
+    val broadcastBlocks = blocks.filter(_.blockId.isInstanceOf[BroadcastBlockId])
```

Comment:
Won't this list always be empty because the listener only tracks information for StreamBlockIds?